lua/autorun/client/cfc_constraint_checker.lua

```diff
@@ -0,0 +1,4 @@
+local NETWORK_NAME = "CFC_ulx-constraint_checker"
```

Comment:
capital letters just get turned into lower letters in net messages, no need to capitalize here. it'd also be simpler to put the string directly into net.Receive

Reply:
actually to be consistent with the rest of the project
it should be `CFC_ULX_ConstraintChecker`

Reply:
And try and think of a name that represent the data the net message contains or the action it performs
perhaps
`CFC_ULX_LogConstraints`

Reply:
👍